test/src/main/webapp/.gitignore

```diff
@@ -1,2 +1,13 @@
-META-INF/maven
-META-INF/MANIFEST.MF
\ No newline at end of file
+/iaf/
+/images/
+/js/
+/jsp/
+/larva/
+/META-INF/maven/
+/META-INF/MANIFEST.MF
+/WEB-INF/
+/body.css
+/favicon.ico
+/ie4.css
+/ie55.css
+/testtool
```

Comment:
Ik denk dat de bedoeling van deze file is om te voorkomen dat kopiën van files die vanuit /iaf/webapp in een root folder terecht komen, niet ingecheckt worden. Dan is het goed om dat als commentaar in de file te vermelden, want je leest zomaar over die slashes aan het begin heen. 
Het lijkt alsof er source files ge-ignored moeten worden voor git, en dat is niet het geval.

Reply:
Ik kan inderdaad wel even een comment toevoegen